src/api/search/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@senecacdot/search-service",
+  "private": true,
+  "version": "1.0.0",
+  "description": "A service for optimizing images",
+  "scripts": {
+    "dev": "nodemon server.js | pino-pretty -c -t",
```

Comment:
`pino pretty` isn't needed now, we do it for you in Satellite.

Reply:
Done